packages/nc-gui/package.json

```diff
@@ -28,6 +28,7 @@
     "jsep": "^1.3.6",
     "locale-codes": "^1.3.1",
     "material-design-icons-iconfont": "^5.0.1",
+    "moment": "^2.29.3",
```

Comment:
<img width="399" alt="Screenshot 2022-06-05 at 09 36 17" src="https://user-images.githubusercontent.com/5435402/172042413-ef6589fe-07ae-4857-9014-11576cf8a67b.png">

Moment is quite heavy. I guess we'd used an alternative. Please check.

Reply:
@wingkwong We already have <a href="https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/package.json">dayjs</a> as a dependency

Reply:
@karishmashuklaa You're right. Will switch to dayjs then.

Reply:
@karishmashuklaa dayjs may not a way to go as it returns `NaN` if we input `minutes: 65`, while momentjs would return `hours: 1` and  `minutes: 5` which is expected. Will think about it.

Reply:
Let's ditch both